### DISCUSSION TITLE
Add all ores to tag c:ores

### DISCUSSION THREAD

**Discussion by Phoupraw** [TechReborn/TechReborn#3010] [Ideas]
Adding all techreborn's ores to tag c:ores will improve compability and comfort with other mods. Take Diggus Maximus as an example, I just need to add only one tag c:ores for all ores, but don't need to add ores one by one. So do other mods' mining machines.

**Comment by Ayutac** [comment]
there are two different kind of ores, these dropping the raw ore and these dropping the ore block. I wonder if there could result troubles from adding them all to c:ores?

Additionally, I would like to know what the c:ores tag is used for, for example? :)

**Reply by Phoupraw** [reply]
As all vanilla ores are in this tag, no matter what they drop, I think all mod ores should be the same.

troubles? what troubles?

This's not necessary and won't bring too many substantial impact, but can make things easier, such as setting blocks for Diggus Maximus, setting filter in Create, Ore Dictionary Export Bus in AE2.

**Reply by Ayutac** [reply]
Just to be clear: you want to add the ore blocks to the tag, not the raw ores, right?

**Reply by Phoupraw** [reply]
> Just to be clear: you want to add the ore blocks to the tag, not the raw ores, right?

Yes, just ores (like silver ore and deepslate silver ore), neither blocks of ores (like block of silver) nor blocks raw ore (like block of raw silver).

**Comment by Ayutac** [comment]
Done with #3015
